src/anm/animation/animation.js

```diff
@@ -356,15 +357,43 @@ Animation.prototype.unsubscribeEvents = function(canvas) {
 };
 
 Animation.prototype.handle__x = function(type, evt) {
+    var anim = this;
     if (events.mouse(type)) {
         var pos = evt.pos;
-        this.each(function(child) {
+        var foundTarget = false;
+        anim.each(function(child) {
```

Comment:
This algorithm confuses me with it's even though it checks bounds first and mouse events are passed only if `options.handleEvents` is `true` and if element is subscribed to them. @mcm69 Probably you have some thoughts on improvement?